includes/REST_API/Stories_Controller.php

```diff
@@ -187,8 +216,10 @@ public function filter_posts_orderby( $orderby, $query ) {
 	 */
 	public function get_items( $request ) {
 		add_filter( 'posts_orderby', [ $this, 'filter_posts_orderby' ], 10, 2 );
+		add_filter( 'posts_join', [ $this, 'filter_posts_join' ], 10, 2 );
```

Comment:
Pasting my feedback from Slack:

There's no need for two separate filters when there's the more convenient `posts_clauses` filter that covers both.